runtime/src/assets.rs

```diff
@@ -11,6 +11,13 @@ lazy_static! {
     static ref ASSET_REGISTRY: Mutex<AssetRegistry> = Mutex::new(AssetRegistry::default());
 }
 
+/// The symbol of Lend Tokens is the symbol of their underlying currency,
+/// prefixed by `Q`. Example: `QDOT` is the `DOT` lend token symbol.
+const LEND_TOKEN_SYMBOL_PREFIX: char = 'Q';
+/// The name of Lend Tokens is the name of their underlying currency,
+/// prefixed by `Lend`. Example: `LendPolkadot` is the `Polkadot` lend token name.
+const LEND_TOKEN_NAME_PREFIX: &str = "Lend";
```

Comment:
Is the name used anywhere? The symbol is used parsing at startup, but just wondering if we also need to add the prefix if the `name()` is used.

Reply:
It's used to make Coingecko API calls, so not directly relevant but thought it's better to have an implementation than to panic